src/ripple/app/ledger/LedgerHistory.cpp

```diff
@@ -384,7 +391,7 @@ void LedgerHistory::handleMismatch (
         log_one (validLedger, (*v)->key(), "built", j_);
 }
 
-void LedgerHistory::builtLedger (Ledger::ref ledger)
+void LedgerHistory::builtLedger (Ledger::ref ledger, Json::Value&& consensus)
 {
```

Comment:
The caller always creates the object just for this call and the call almost always needs to store it. Passing this way avoids a copy of a large, deep object.